molecule/resources/tests/verify.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Verify Linux
+  hosts: rhel_family:debian_family
+  tasks:
+  - name: Check Zookeeper service status
+    service:
+      name: 'zookeeper'
+    check_mode: true
+    register: zookeeper_status
```

Comment:
Why we don't use simple `service_facts` module?

Reply:
There are several reasons for this:
1. I want to stick to the same approach for Linux and Windows, i.e. **service** module for Linux and **win_service** for Windows.
2.  **service_facts** gives not very accurate results, for example, in the case described in this issue https://github.com/lean-delivery/ansible-role-zookeeper/issues/16 **service_facts** reports that service is running, but it is not.